backend/auth-service/src/main/kotlin/com/nosql/authservice/common/exception/NotFoundException.kt

```diff
@@ -0,0 +1,13 @@
+package com.nosql.authservice.common.exception
+
+import com.nosql.authservice.common.exception.enumerator.ApplicationExceptionCodeEnum.NOT_FOUND_ERROR
+import org.springframework.http.HttpStatus
+
+open class NotFoundException(
+    status: HttpStatus = HttpStatus.NOT_FOUND,
```

Comment:
maybe use `HttpStatus` only in the `ApplicationException` constructor?

Reply:
fixed